packages/components/search/src/components/AllFiltersSelect.vue

```diff
@@ -0,0 +1,126 @@
+<template >
```

Comment:
change file names so consistent with kronos. If kronos uses param case for file names do that for all.  all-select-filter ....

Reply:
No need to change name. When we use this component we use small later name. like: all-filter-select